### DISCUSSION TITLE
sql lite crash again in again try to reload reinstall nothing working

### DISCUSSION THREAD

**Discussion by Cafesail** [pawelsalawa/sqlitestudio#3821] [Q&A]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
What is your operating system?
